h/sentry.py

```diff
@@ -14,6 +14,11 @@
 
 from h import __version__
 
+PROCESSORS = (
+    'raven.processors.SanitizePasswordsProcessor',
+    'raven.processors.RemovePostDataProcessor',
```

Comment:
`git log -S RemovePostDataProcessor` will give you that information straight away ;)